src/test/java/ru/training/at/hw2/ex1/EpamDemoWebPageTests.java

```diff
@@ -0,0 +1,112 @@
+package ru.training.at.hw2.ex1;
+
+import java.util.List;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.WebDriverWait;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+public class EpamDemoWebPageTests extends WebPageTestsCommonConditions {
+
+    @Test
+    public void demoPageTests() {
+        //1.Open test site by URL
+        driver.get(HOMEPAGE_URL);
+
+        //2. Assert Browser title
+        String expectedTitle = "Home Page";
+        String actualTitle = driver.getTitle();
+        Assert.assertEquals(actualTitle, expectedTitle);
+
+        //3.Perform login
+        new WebDriverWait(driver, WAIT_TIMEOUT_SECONDS).until(ExpectedConditions
+            .elementToBeClickable(By.className("profile-photo"))).click();
+
+        new WebDriverWait(driver, WAIT_TIMEOUT_SECONDS).until(ExpectedConditions
+            .elementToBeClickable(By.id("name"))).sendKeys(LOGIN_NAME);
```

Comment:
Linebreaks are hard to read. Compare to the following
```suggestion
        new WebDriverWait(driver, WAIT_TIMEOUT_SECONDS)
            .until(ExpectedConditions.elementToBeClickable(By.id("name")))
            .sendKeys(LOGIN_NAME);
```

Reply:
Corrected